Cloudflare/access_requests/ingest/parser.yml

```
@@ -0,0 +1,38 @@
+name: Cloudflare Access Requests
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.CreatedAt}}'
+          client.ip: '{{json_event.message.IPAddress}}'
+          event.category: ["network", "authentication"]
+          event.dataset: "access_request"
+          event.kind: 'event'
```

I suggest adding some fields
```suggestion
          event.kind: 'event'
          event.dataset: "access_requests"
          event.type: ["info"]
          observer.type: "proxy"
          observer.vendor: "Cloudflare"
```